### ISSUE TITLE
adjust colors of filtered numeric facet

### ISSUE THREAD

**Issue by marwahaha** [OpenRefine/OpenRefine#3319]
I taught a workshop on OpenRefine today, and some learners could not see the dark-blue on light-blue histograms when filtering data in a numeric facet.

### Proposed solution
Add a color choosing menu that adjusts the "filtered histogram color" (and perhaps the initial histogram color).

### Alternatives considered
I don't know of another way in OpenRefine to access a histogram of filtered data, or to compare the distribution 

### Additional context
The color contrast was not strong enough for them:

![image](https://user-images.githubusercontent.com/2541209/98302369-1a6c4200-1f71-11eb-8797-78305cb5f6bc.png)

**Comment by tfmorris** [comment]
Thanks for the suggestion. While we could add color pickers or theming, I think we should start by making the defaults meet the accessibility guidelines with a minimum contrast ratio of 3:1 (4.5:1 recommended). Lighthouse (accessibility checker) says that the light blue text on lighter blue background (ie "change" & "reset") doesn't meet this minimum either.

The colors for the bottom histogram are: 
-  unselected (muted) 174, 199, 251 #aec7fb
- selected 116, 162, 231 #74a2e7  - contrast ratio 1.53:1

and for the unselected (right hand side) of the top histogram:
- unselected x2 133, 145, 171 #8591ab
- unselected in this facet, selected in other 104, 127, 161 #687fa1 - contrast ratio 1.29:1

Neither contrast ratio is anywhere near acceptable and this is exacerbated for some of the text by the combination of small font size and poor contrast ratio.

We should probably do an accessibility review of the entire UI. I'm sure there are lots more items which need attention.

In the meantime, one hint which might help is try and be as square to the display as possible. Flat panels in particular are very finicky about off-axis viewing which can make contrast issues worse.
